components/compliance-service/inspec/cli.go

```diff
@@ -234,64 +241,71 @@ func Archive(profilePath string, outputPath string) error {
 	return nil
 }
 
-func getInspecError(serr string, err error, target *TargetConfig, timeout time.Duration) *Error {
+func getInspecError(stdOut string, stdErr string, err error, target *TargetConfig, timeout time.Duration) *Error {
```

Comment:
this change makes me sad b/c it means that we're somehow getting the error in stdout instead of stderr.....but i also want the pipeline to get unblocked AND we're all very uncertain of how this started happening...

Reply:
I found that config like this would force the logging to stderr:

Reply:
i...wonder if this has anything to do with the fact that we haven't changed our code to use reporter (instead of the original "format") ?https://github.com/chef/automate/blob/master/components/compliance-service/inspec/cli.go#L109